.github/workflows/scripts/build_push.sh

```diff
@@ -54,6 +54,9 @@ for MEGA_SVC in $1; do
             docker_build ${IMAGE_NAME}-ui docker/Dockerfile
             if [ "$MEGA_SVC" == "ChatQnA" ];then
                 docker_build ${IMAGE_NAME}-conversation-ui docker/Dockerfile.react
+            fi
+             if [ "$MEGA_SVC" == "CodeGen" ];then
```

Comment:
fix format
![image](https://github.com/user-attachments/assets/6fdfb5c1-8a4d-407f-8dc3-c704678b49dd)

Reply:
fixed it